x-pack/libbeat/management/generate.go

```diff
@@ -68,7 +68,11 @@ func (r *TransformRegister) Transform(cfg *proto.UnitExpectedConfig, agentInfo *
 // that can later be formatted into the reloader's ConfigWithMetaData and sent to an indvidual beat/
 // This also performs the basic task of inserting module-level add_field processors into the inputs/modules.
 func CreateInputsFromStreams(raw *proto.UnitExpectedConfig, inputType string, agentInfo *client.AgentInfo) ([]map[string]interface{}, error) {
-	inputs := make([]map[string]interface{}, len(raw.Streams))
+	// should this be an error?
```

Comment:
I don't know that an input without streams is strictly an error in the agent policy, but it is definitely suspicious for a unit config sent to a Beat.

Reply:
Yah, I was just worried about breaking weird edge cases that were working before.